.drone.jsonnet

```diff
@@ -17,6 +17,6 @@ local Pipeline(os, arch, version) = {
 };
 
 [
-    Pipeline("linux", "arm",   "1.4"),
-    Pipeline("linux", "arm64", "1.4")
+    Pipeline("linux", "arm",   "1.5"),
```

Comment:
There is no build for Julia v1.5 on armv7l.  The latest ones are 1.3.1 and v1.4.1

Reply:
To be clear, this file is not used by Drone directly, only `.drone.yml` matters, and that one didn't change.  This is useful to generate the YAML file automatically though, which is much more cumbersome to do when you want to support multiple Julia versions and/or architectures.  Note also v1.4.1 != v1.4, because there has been v1.4.2